config/default/user.role.authenticated.yml

```diff
@@ -27,8 +26,8 @@ permissions:
   - 'access shortcuts'
   - 'delete own files'
   - 'download media'
+  - 'opt-in or out of google analytics tracking'
```

Comment:
Please remove this line. This config is ignored, but it's better to avoid adding unrelated changes.

Reply:
Removed.